client/src/main/java/io/split/client/SplitClientImpl.java

```diff
@@ -94,30 +95,21 @@ private String getTreatment(String matchingKey, String bucketingKey, String spli
 
             long start = System.currentTimeMillis();
 
-            TreatmentLabelAndChangeNumber result = null;
-            try {
-                result = getTreatmentWithoutExceptionHandling(matchingKey, bucketingKey, split, attributes);
-            } catch (ChangeNumberExceptionWrapper e) {
-                result = new TreatmentLabelAndChangeNumber(Treatments.CONTROL, EXCEPTION, e.changeNumber());
-                _log.error("Exception", e.wrappedException());
-            } catch (Exception e) {
-                result = new TreatmentLabelAndChangeNumber(Treatments.CONTROL, EXCEPTION);
-                _log.error("Exception", e);
-            } finally {
-                recordStats(
-                        matchingKey,
-                        bucketingKey,
-                        split,
-                        start,
-                        result._treatment,
-                        "sdk.getTreatment",
-                        _config.labelsEnabled() ? result._label : null,
-                        result._changeNumber,
-                        attributes);
-            }
+            TreatmentLabelAndChangeNumber result = getTreatmentResultWithoutImpressions(matchingKey, bucketingKey, split, attributes);
 
-            return result._treatment;
+            recordStats(
```

Comment:
Fair question. I can change the catch(Exception) to catch throwable - since that seems to be the intention anyway